test/Elastic.Apm.Tests.MockApmServer/Elastic.Apm.Tests.MockApmServer.csproj

```
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;net50</TargetFrameworks>
```

```suggestion
    <TargetFramework>net5.0</TargetFramework>
```

Propose we remove netcoreapp2.1 altogether, and update test projects using it to net5.0 too, which are

- Elastic.Apm.AspNetFullFramework.Tests
- Elastic.Apm.StartupHook.Tests

And update the include condition on 
- Elastic.Apm.Tests
to net5.0.